Probesys/bileto
# i18n: Translate the search form tabs

## Related issue(s)



N/A

## Changes



See commits

## How to test manually



- switch the interface to English
- go to the tickets list → check that the search tabs are translated

## Checklist
